x-pack/plugins/apm/public/components/app/Correlations/index.tsx

```diff
@@ -19,33 +19,41 @@ import {
 } from '@elastic/eui';
 import { useHistory } from 'react-router-dom';
 import { EuiSpacer } from '@elastic/eui';
-import { enableCorrelations } from '../../../../common/ui_settings_keys';
+import { enableSignificantTerms } from '../../../../common/ui_settings_keys';
 import { useApmPluginContext } from '../../../context/apm_plugin/use_apm_plugin_context';
 import { LatencyCorrelations } from './LatencyCorrelations';
 import { ErrorCorrelations } from './ErrorCorrelations';
 import { useUrlParams } from '../../../context/url_params_context/use_url_params';
 import { createHref } from '../../shared/Links/url_helpers';
+import { useLicenseContext } from '../../../context/license/use_license_context';
 
 export function Correlations() {
   const { uiSettings } = useApmPluginContext().core;
   const { urlParams } = useUrlParams();
+  const license = useLicenseContext();
+  const hasValidPlatinumLicense =
+    license && license.isActive && license.hasAtLeast('platinum');
```

Comment:
We have a helper in common for this. It's in a service_map directory, though perhaps we could put it somewhere more generic. Not required for this but would be a good refactor.

https://github.com/smith/kibana/blob/890a9ac366ca5ba82e1fe1d67616de36cc44e68c/x-pack/plugins/apm/common/service_map.ts#L64-L66

Reply:
Honestly, I'm not sure how much that helper gives us. It's fairly concise as is, isn't it?

Reply:
otoh, I see we've duplicated the same helper in this very PR. So I agree it's better to share than duplicate.